app/src/main/java/org/mozilla/fenix/components/metrics/GleanMetricsService.kt

```diff
@@ -497,13 +498,6 @@ class GleanMetricsService(private val context: Context) : MetricsService {
         // can handle events being recorded before it's initialized.
         gleanInitializer = MainScope().launch {
             Glean.registerPings(Pings)
```

Comment:
Ideally, you would move this to before the new call site of `Glean.initialize` as well.  In practice, it currently only affects custom pings containing events, though, and I don't think Fenix has any custom pings containing events.